app/controllers/authors_controller.rb

```diff
@@ -37,7 +37,7 @@ def settings
       @scroll_to_guestbook = true
     end
     @guestbook_entries = @author.guestbook_entries.where(spam: false)
-    @posts = @author.posts
+    @posts = @author.posts.sort { |a, b| b.created_at <=> a.created_at }
```

Comment:
Does ActiveRecord level sorting work here? `@author.posts.order('DESC')` or something like that.

Reply:
fixed! :) @mobitar